### PR TITLE
Adds new traitor "bundle" the junk box.

### PR DESCRIPTION
**Pull request by itseasytosee**
## About The Pull Request

Adds a one TC box to the pointless (badassery) section to the traitor uplink that contains syndicate "junk" Basically it picks a random unique gimmicky kits/item that I made along with some fluff text in a box under the guise of being some odd junk cluttering up some syndie listening base or outpost.
The possible items are as follows,

**The bastard's knife**- A knife on par with an unsharpened chef knife in terms of damage. However, it is capable of slitting throats at the max speed, on par with cult blades and energy swords (3 seconds max) also adds a var that can be used to modify throat-cutting speed.

**Syndie PJs** - Comes with a bedsheet. Has slight damage resistance against lasers. The gimmick is they allow you to recover from much more grievous wounds by sleeping and increase the speed at which health is recovered.

**Fake mindshield implant** (is now its own item for 3 TC)- Has no effect other than showing up as mindshield at first glace to sec huds. Does not fool beepskys or guns.

**Syndie Snack Box** -Filled with some stale food. It contains a plastitanium fork which has damage still on par with a fork but has 100% armor-piercing and can stab people through masks.

I hope there are more in the future and I can think of a better gimmick for the snack box.

## Why It's Good For The Game
Gives me and others a place to put small ~1 TC gimmicky or joke syndie items without cluttering up 
the uplink.
Also supplies something for people sitting on a tc or two to buy without any direction
I hope that in the future I and others can expand on it. I would love to hear suggestions for items and bundles

## Changelog
:cl:
add: A item to the syndie uplink. Syndicate junk found in the pointless section of your uplink. Gives a relatively unpowerfull, unique, and gimmicky item for only 1 tc
add: Fake mindshield implant. Purchasable for 3 TC only manages to fool security huds and nothing else.
/:cl:

### CODE REVIEW

**Comment by Archanial** [comment]
Fake mindshield for 1 tc, are you sure about that?

**Comment by Shadowflame909** [comment]
@Archanial The gimmick is it's shittily fake, and only fools sec you're speed past.

Probably made with crayon art.

**Comment by Anonmare** [comment]
The mindshield is going to be incredibly strong when combined with a chamo kit set to security officer and a purloined flash/baton, though it's only potentially 1 TC so it *might* be fine. People don't think twice at sec randomly arresting people after all.

**Comment by tralezab** [comment]
Fake mindshield should just be an actual item LMAO

**Comment by AdamElTablawy** [comment]
Both the fake mindshield and the knife that lets you slit throats sound like legit decent items

**Comment by HomocidlePancakes** [comment]
Make the fork bypass eyewear. Вилкой в глаз или в ж....

**Comment by XDTM** [comment]
Code-wise the fake implant uses the same icon as normal implants, which makes it nearly impossible to tell from the real deal. If it was in the uplink, it'd probably cost a few more TC, but i'd avoid making it so easy for traitors to pose as security, since it would heavily erode the trustworthiness of mindshields.

**Comment by Archanial** [comment]
You should make fake mindshield its own item for 2-3 tc, and replace it with something (maybe fake bomb?).

**Comment by rdragan** [comment]
The only way a traitor should be able to get a mindshield, even a fake one, is from the armory or cargo, like everyone else. Having it in the uplink for any amount is silly. Having it for 1 TC is insanity.

**Comment by knacker48** [comment]
The only way I see to balance it is to make a lot more items so much less consistent to get or nerfing it in a way that lets people know that its fake

**Comment by tralezab** [comment]
rdragan fake mindshields are pretty niche

**Comment by itseasytosee** [comment]
Ill make a new sprite for the mindshield if you guys think its that OP. Maybe an unmoving sprite in contrast to the normal mineshield's flashing?

**Comment by tralezab** [comment]
we think it's good in that it's a great idea for a traitor item, not that it's OP. that is a terrible idea

**Comment by PKPenguin321** [comment]
Yeah just make it it's own item

**Comment by Archanial** [comment]
You could add a bomb that explodes into harmless glitter to your original bundle.
Also what about some check that would make you unable to fake-mindshield a headrev?

**Comment by optimumtact** [comment]
Things this server needs

crew content, new ruins, new maps, fixes and features.

Things this server doesn't need:
New antag items that are useless.

**Comment by itseasytosee** [comment]
The idea is that the items are not useless but extremely gimmicky each one has their own minor purpose.

**Comment by itseasytosee** [comment]
@Archanial Sounds like a fun idea. Also despite how unlikely it may be its a good idea to check for rev minds.

**Comment by optimumtact** [comment]
they're useless

**Comment by itseasytosee** [comment]
Strongly disagree

File: code/game/data_huds.dm [hunk 1]
```diff
@@ -246,6 +246,11 @@
 		var/icon/IC = icon(icon, icon_state, dir)
 		holder.pixel_y = IC.Height() - world.icon_size
 		holder.icon_state = "hud_imp_loyal"
+	if (HAS_TRAIT(src, TRAIT_FAKE_MIND_SHIELD))
```
Comment (rdragan) [inline code/game/data_huds.dm]: You should check for this at line 244 along with the regular mindshield trait.

**Comment by ExcessiveUseOfCobblestone** [comment]
I'm personally not merging any more "items that directly kill BUT in a different flavor" for the uplink. I think there's plenty and am tired of giving murderboners more tools for their cruddy trade.

**Comment by cacogen** [comment]
tired of these autistic goon-tier prs

**Comment by itseasytosee** [comment]
Fair. I'll probably close this tonight and maybe bring back the fake mindshield sometime because of its reception.

**Pull request closed without merging**